src/widgets/ProfileWidget/ProfileWidget.scss

```
@@ -0,0 +1,44 @@
+@import "src/styles/index";
+
+.profile-widget {
+
+    &__button-group-container {
+        display: block;
+        margin-top: 1.625rem;
+
+        @include for-size(phone-only) {
+            margin-top: 9.125rem;
+        }
+
+        @include for-size(desktop-up) {
+            display: flex;
+            justify-content: center;
+        }
+    }
+
+    &__button-group {
+        display: flex;
+        border: solid 1px var(--c-grey);
+        border-radius: 4px;
+
+        @include for-size(desktop-up) {
+            width: 19.5rem;
+            transform: translateY(-2.56rem);
+        }
+    }
+
+    &__button-group__button {
+        width: -webkit-fill-available;
+        height: 2rem;
+        font-weight: 700;
+        font-size: .75rem;
+        line-height: 1.5rem;
+        color: var(--c-light-grey);
+        background: var(--c-dark);
+
+        &--is-active {
+            background-color: var(--c-grey);
+            color: #fff;
+        }
+    }
+}
```

```suggestion
}

```

Newline at end of file